editions/2015-02.md

```diff
@@ -0,0 +1,61 @@
+
+
+### [Growstuff](http://www.growstuff.org/)
+
+Growstuff is ‘a community of food gardeners who are building an open source platform to help you learn about growing food, track what you plant and harvest, and swap seeds and produce with other gardeners near you.’
+
+We love this one because it’s a practical open source project helping people grow food and community. It helps keep information held in plants and seeds open by making it easy to share. Growstuff releases the data it collects under open licenses, giving power to it’s developers as well as growing a community of gardeners. It promotes pair programming as the norm.
+
+This is a very active project. Get involved as a volunteer to help test, write for [the Growstuff blog](http://blog.growstuff.org/), [contribute code to the application](https://github.com/growstuff) or just dig in, grow some food and share your seeds and data.
+
+
+### [Nick Evershed injects some context into the news](http://www.theguardian.com/news/datablog/2015/feb/14/tony-abbott-is-not-alone-in-using-the-word-holocaust-to-score-political-points)
+
+After Tony Abbott apologised for saying Labor created a “holocaust” in defence industry jobs, [Nick Evershed](https://twitter.com/nickevershed) used the [OpenAustralia API](http://www.openaustralia.org.au/api/) to provide more context about the use of the term in Parliament.
+
+
+### [Hackfarming Blood Buddies](https://adactio.com/journal/8352)
+
+[Jeremy Keith](https://adactio.com/) takes us through the process of planning, building and deploying a hack to help more people donate blood at [Clearleft’s](http://clearleft.com/) Hackfarm getaway this year.
+
+Their 2012 Hackfarm resulted in a prototype for [a personal political opinion tracker](http://hackfarm.org/product)—a few people at the [OpenAustralia Foundation meetup](http://www.meetup.com/OpenAustralia-Foundation/) have suggested similar concepts for projects they want to work on.
+
+
+### [Local events for Internation Open Data Day](http://au.okfn.org/2015/02/10/australian-activities-on-international-open-data-day-21-february/)
+
+This Saturday, 21st Feb, is [International Open Data Day](http://opendataday.org/). The Open Knowledge Foundation Australia have done a wrap-up of events, including a day long workshopping session hosted by [Code for Australia](http://www.codeforaustralia.org/) in [Sydney tomorrow](http://www.eventbrite.com.au/e/international-open-data-day-sydney-tickets-15462067460). It is a free event and registrations are still open if you can make it.
```

Comment:
so I think of a wrap up as being finishing something up or summarizing something which is complete - maybe that's a newer usage? also the use of 'have done' - past tense for a future event - I find it confusing. If I'm not alone in this - change "have done a wrap-up". Also let's change "a day long workshopping session" to "a day of workshops" or a "day long workshop"  if it is one long workshop? Also remove 'if you can make it' it's not needed.